examples/kubeval/imperative/.expected/config.yaml

```diff
@@ -0,0 +1,8 @@
+exitCode: 1
+testType: eval
+image: gcr.io/kpt-fn/kubeval:unstable
+args:
+  strict: 'true'
+stdErr:
+  - "[ERROR] Additional property templates is not allowed in object 'v1/ReplicationController//bob' in file resources.yaml in field templates"
```

Comment:
With structured results design, there's a change in plan for testing. We should have a few e2e tests in the kpt repo to test the presentation layer. But otherwise, we should only test the structured output which should provide all the information we want to test the functions robustly (depend on APIs not presentation layer). In particular, packages in the catalog repo shouldn't test the presentation layer. Otherwise, you'd get brittle change detector tests:

https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html

Changing the presentation layer in the future should involve updating just a few tests.

Reply:
@frankfarzan I remember we have that discussion in another PR. Are you suggesting hold this PR until that change lands?
@Shell32-Natsu What is the status? Any ETA?

Reply:
Updated the PR to only test structured results

Reply:
>@droot is adding support GoogleContainerTools/kpt#1923

> Can you coordinate with him?

Looking at the time stamp of this comment, I guess you intended to publish last week but forgot.